docs/designers-developers/designers/design-patterns.md

```diff
@@ -2,7 +2,7 @@
 
 ## Basic Editor Interface
 
-Gutenberg’s general layout uses on a bar at the top, with content below.
+The Block Editor’s general layout uses on a bar at the top, with content below.
```

Comment:
I believe the standard is not to refer to the block editor as proper thing, so it should be lowercase by default, unless in a title.

```suggestion
The block editor’s general layout uses on a bar at the top, with content below.
```